sql/rpl_slave.cc

```diff
@@ -6344,10 +6344,9 @@ bool mts_checkpoint_routine(Relay_log_info *rli, bool force) {
   /* TODO:
      to turn the least occupied selection in terms of jobs pieces
   */
-  for (Slave_worker **it = rli->workers.begin(); it != rli->workers.begin();
-       ++it) {
-    Slave_worker *w_i = *it;
-    rli->least_occupied_workers[w_i->id] = w_i->jobs.len;
+  for(size_t i= 0; i < rli->get_worker_count(); i++) {
```

Comment:
Please format all the `.h` & `.cc` files in this commit using `clang-format-10`.